playbook/package.json

```diff
@@ -44,6 +44,7 @@
     "@babel/preset-react": "^7.12.5",
     "@floating-ui/react-dom": "^1.0.0",
     "@floating-ui/react-dom-interactions": "^0.9.1",
+    "@fortawesome/fontawesome-free": "^6.2.1",
```

Comment:
Why is this added? I would have expected to see:

`"@fortawesome/fontawesome-pro": "^6.2.1"`...

Reply:
The idea was that the website will have pro installed while playbook as a design system will ship with free as the default. Users have to install the pro version in their codebases themselves if they wish to use it.